tests/generic_config_updater/test_bgp_prefix.py

```diff
@@ -0,0 +1,221 @@
+import logging
+import pytest
+import re
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.generic_config_updater.gu_utils import apply_patch, expect_op_failure, expect_op_success
+from tests.generic_config_updater.gu_utils import generate_tmpfile, delete_tmpfile
+from tests.generic_config_updater.gu_utils import create_checkpoint, delete_checkpoint, rollback_or_reload
+
+pytestmark = [
+    pytest.mark.topology('t0'),  #TODO: t0?
```

Comment:
>#TODO: t0 [](http://example.com/codeflow?start=33&length=9)

The TODO comment does not make sense. You already covered t0.

Reply:
fixed